Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/Microsoft.MixedReality.Toolkit.SU.asmdef

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We need to add this assembly to the default link.xml contents.

Reply:
Please see  Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityToolkitPreserveSettings.cs